src/views/Portal/components/ApplicationForm/Connected/index.view.tsx

```diff
@@ -1,64 +1,74 @@
-  // placeholder
-  const handleChange = () => {}
-  const errors = ""
+  const { connectedFormData, setConnectedFormData } = useApplication()!
+  const handleChange = (event: any) => {
+    const name = event.target.getAttribute("name")
+    if (Object.keys(connectedFormData).includes(name)) {
+      const copy = { ...connectedFormData, [name]: event.target.value }
+      setConnectedFormData(copy)
+    }
+  }
   return (
     <div className='connected-page'>
       <div className='connected-page__container'>
         <div className='connected-page__container--title'>Get Connected</div>
-        <div className='connected-page__container--info'>
-          Want to share your resume with our sponsors? Fill out our resume drop
-          form here.
-        </div>
-        <input type='file' />
+        <FileUpload
+          question='Want to share your resume with our sponsors? Upload your pdf resume here.'
+          errorMessage={connectedFormData.resumeErr}
+          handleChange={setConnectedFormData}
+          name='resume'
+          className='none'
+        />
         <div className='connected-page__container--body'>
           <TextField
             name='LinkedIn'
             label='LinkedIn'
             maxLength={50}
             handleChange={handleChange}
-            fieldState={undefined}
-            errorMessage={errors}
+            fieldState={connectedFormData.linkedin}
+            errorMessage={connectedFormData.linkedinErr}
           />
           <TextField
             name='GitHub'
             label='GitHub'
             maxLength={100}
             handleChange={handleChange}
-            fieldState={undefined}
-            errorMessage={errors}
+            fieldState={connectedFormData.github}
+            errorMessage={connectedFormData.githubErr}
           />
           <TextBox
-            label='Reference'
+            // className='connected-page__container--textbox'
+            label='cruzCoins'
             name='How did you hear about CruzHacks? If someone specific referred you, let us know their name so they can earn CruzCoins!'
             maxLength={100}
             handleChange={handleChange}
-            fieldState={undefined}
-            errorMessage={errors}
+            fieldState={connectedFormData.cruzCoins}
+            errorMessage={connectedFormData.cruzCoinsErr}
           />
-          <RadioForm
+          {/* <RadioForm
             question='Are you part of CruzCoins? And if not, would you like to register?'
             name='CruzCoins Interest'
             inputs={[{ label: "Yes" }, { label: "No" }]}
             handleChange={handleChange}
             checkedState={() => {}}
             errorMessage={errors}
-          />
+          /> */}
+          <div className='connected-page__container--info'>
+            Are you part of CruzCoins? And if not, would you like to register?
+          </div>
           <TextBox
             label='OtherInfo'
```

Comment:
```suggestion
            label='anythingElse'
```


We also need to add `name={label}` to the TextBox so `handleChange` can work: https://github.com/CruzHacks/cruzhacks-2022-website/blob/fe10e5ddf75b2b37ca3cc80c2a9daca9a473c0ba/src/views/Portal/components/FormInputs/TextBox/index.tsx#L16

Reply:
Similar to how we did it here: https://github.com/CruzHacks/cruzhacks-2022-website/blob/fe10e5ddf75b2b37ca3cc80c2a9daca9a473c0ba/src/views/Portal/components/FormInputs/TextField/index.tsx#L22